test/mip_test.c

```diff
@@ -2,15 +2,17 @@
 #include "mongoose.c"
 
 static void test_queue(void) {
-  uint8_t buf[sizeof(size_t) + 5];
+  static uint8_t buf[sizeof(size_t) + sizeof(uint16_t) + 3 ]; // fit 1 element but not 2
```

Comment:
why static? we call this function only once, there is no point to memoise it

Reply:
I commented about it above, it makes it easier to step the code, otherwise those are optimized out.
Thought you might like to check.
Can go back to auto variables